test/consul.js

```diff
@@ -346,57 +346,6 @@ describe('Consul', () => {
     });
   });
 
-  it('resolves multiple tags as separate services', () => {
-    const consul = generateConsulStub();
-
-    consul.initialize();
-    consul.mock.emitChange('catalog-service', {consul: ['production', 'development']});
-    consul.mock.emitChange('consul-production', [{
-      Service: {Address: '127.0.0.1'}
-    }]);
-    consul.mock.emitChange('consul-development', [{
-      Service: {Address: '10.0.0.0'}
-    }]);
-
-    return Promise.resolve(consul.properties.consul).should.eventually.eql({
-      'consul-production': {
-        cluster: 'production',
-        addresses: ['127.0.0.1']
-      },
-      'consul-development': {
-        cluster: 'development',
-        addresses: ['10.0.0.0']
-      }
-    });
-  });
-
-  it('avoids resolving similar tags to the same service', () => {
```

Comment:
I think I'll set this as a "pending" test. We can fill in the implementation when we include tag parsing.